app/src/test/java/com/example/trainingintive/viewmodels/SplashViewModelTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.trainingintive.viewmodels
+
+import com.example.trainingintive.TestSchedulersProvider
+import com.example.trainingintive.navigators.SplashNavigator
+import com.example.trainingintive.util.SplashScreenEvent
+import io.mockk.justRun
+import io.mockk.mockk
+import io.mockk.verify
+import io.reactivex.rxjava3.plugins.RxJavaPlugins
+import io.reactivex.rxjava3.schedulers.TestScheduler
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.util.concurrent.TimeUnit
+
+class SplashViewModelTest {
+
+    private lateinit var viewModel: SplashViewModel
+    private val navigator = mockk<SplashNavigator> {
+        justRun { sendEvent(any()) }
+    }
+    private val schedulersForTest = TestSchedulersProvider()
+
+    @Before
+    fun setUp() = RxJavaPlugins.reset()
+
+    @After
+    fun tearDown() = RxJavaPlugins.reset()
+
+    @Test
+    fun `after init viewmodel should send DisplayUserScreen event`() {
+        val testScheduler = TestScheduler()
+        RxJavaPlugins.setComputationSchedulerHandler { testScheduler }
+        viewModel = SplashViewModel(navigator, schedulersForTest)
+        testScheduler.advanceTimeBy(5, TimeUnit.SECONDS)
+
+        verify() { navigator.sendEvent(SplashScreenEvent.DisplayUserScreen) }
```

Comment:
You can (and you should :)) drop unecessary brackets, simply call `verify { ... }`.